app/assets/javascripts/application.js

```diff
@@ -11,6 +11,7 @@
 // about supported directives.
 //
 //= require rails-ujs
+// require activestorage
```

Comment:
I think you're missing a `=` here?

Reply:
left it out intentionally, [`active_storage/engine`](https://github.com/fidor/fidor_api_demo/pull/34/files#diff-b1fe55db50c712fef0673345e5b9c0d9R8) isn't load in this app.

Reply:
:+1: